ast/ast.go

```diff
@@ -1445,6 +1445,29 @@ type CastNumValue struct {
 //
 // ================================================================================
 
+// Options is generic OPTIONS clause node without key and value checking.
+//
+//	OPTIONS ({{.Records | sqlJoin ","}})
+type Options struct {
+	// pos = Options
+	// end = Rparen + 1
+
+	Options token.Pos // position of "OPTIONS" keyword
+	Rparen  token.Pos // position of ")"
+
+	Records []*OptionsDef // len(Records) > 0
+}
+// OptionsDef is single option definition for DDL statements.
```

Comment:
Insert a newline here.

```suggestion
}

// OptionsDef is single option definition for DDL statements.
```